package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "no-hassle",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
See PR description for explanation. Short of it is: treating this as a "fix" to the major version `3.0.0`.